packages/components/table/stories/table.stories.js

```diff
@@ -1,12 +1,110 @@
-import { storiesOf, html, select } from '@open-wc/demoing-storybook';
-import { withKnobs, array } from '@storybook/addon-knobs';
+import { storiesOf, html, select, text, array } from '@open-wc/demoing-storybook';
+import { withKnobs } from '@storybook/addon-knobs';
 
 import '@tradeshift/elements.table';
+import readme from '../README.md';
+
+/**
+ * Decodes the string with encoded html characters.
+ * @param {string} input
+ */
+function htmlDecode(input) {
+	const e = document.createElement('textarea');
+	e.innerHTML = input;
+	// handle case of empty input
+	return e.childNodes.length === 0 ? '' : e.childNodes[0].nodeValue;
+}
 
 storiesOf('ts-table', module)
 	.addDecorator(withKnobs)
-	.add('default', () => {
-		const cols = array('cols', [
+	.add(
+		'default',
+		() => {
+			const columnsDefaultConfig = JSON.stringify([
+				{
+					property: 'name',
+					size: 'large',
+					value: 'Context'
+				},
+				{
+					visibility: 'desktop-only',
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal 8 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS1192)